texts/epn_paper.md

```diff
@@ -0,0 +1,19 @@
+# Let's talk online
```

Comment:
That's a tentative title, opening a discussion of the final one.

Reply:
This title I OK, but if anyone has better idea, please add.

Reply:
This is similar, but:
```suggestion
# Let's take this discussion online
```
or 
```suggestion
# Let's continue this discussion online
```

Reply:
I like Evert's suggestion.